deepchecks/vision/metrics_utils/scorers.py

```diff
@@ -140,9 +140,15 @@ def get_scorers_dict(
                     raise DeepchecksNotSupportedError(
                         f'Unsupported metric: {name} of type {type(metric).__name__} was given.')
                 scorers[name] = converted_met
+            elif isinstance(metric, t.Callable):
```

Comment:
Added CustomMetric

Reply:
Yep thats reasonable

Reply:
Would add additional validation for this if (validate that the callable is in the required format, maybe this can be done in DeepchecksScorer now that i think about it

Reply:
As this is a problem with tabular as well, would do in PR unrelated to #2211

Reply:
sure